kaldb/src/main/java/com/slack/kaldb/logstore/LogDocumentBuilderImpl.java

```diff
@@ -370,4 +372,9 @@ public Document fromMessage(LogMessage message) throws JsonProcessingException {
     }
     return doc;
   }
+
+  @Override
+  public Map<String, LuceneFieldDef> getSchema() {
+    return Collections.emptyMap();
```

Comment:
This should have a comment explaining why an empty map is appropriate here.

Reply:
Fixed.